docs/dasharo-menu-docs/overview.md

```diff
@@ -10,10 +10,125 @@ subsections you will get to know:
+  It may also contain informational menus like Driver Health Manager, Network
+  Device List and others.
+* [Dasharo System Features](dasharo-system-features.md)- contains submenus for
+  features specific to Dasharo products and Dasharo supported platforms
+* [One Time Boot](#one-time-boot) - allows to choose which boot entry to
+  execute. It simply lists all available boot options and allows to select one
+  the same way as [Boot Manager Menu](#boot-manager-menu)
+* [Boot Maintenance Manager](boot-maintenance-mgr.md) - allows to manipulate
+  various UEFI standard variables responsible for console and boot options. One
+  may choose which devices should be used for input and output, choose to boot
+  an arbitrary file or modify the boot options and boot order.
+
+NOTE: not all submenus may be available on your platform.
+[Contact Dasharo Team](mailto:contact@dasharo.com) for more information and
+possible feature extension of your platform.
+
+### User Password Management
+
+User Password Management menu allows one to set firmware setup admin password:
+
+![](/images/menus/password_mgr.jpeg){ class=center" }
+
+The password must be a strong one. The requirements are described on the help
+section on the right side of the menu page. If the password is too weak it will
+not be accepted. The page shows the current status of `Admin Password` if it
+was provisioned or not.
+
+To disable the password, enter the `User Password Management` menu again and
+request to set new password. You will be asked to enter current password. When
+a window to provide new password pops up, leave the field empty and confirm it
+twice with `ENTER`. The password installation status on the menu page should
+change to `Not Installed`.
+
+### One Time Boot
+
+When setup application is entered, one of the menus is called `One Time Boot`.
+As the name suggests it allows to override the boot just one time (not
+permanently). The usage principles are the same as for [Boot Manager Menu](#boot-manager-menu).
+
+Example view of `One Time Boot` submenu:
+
+![](/images/menus/one_time_boot.jpeg){ class="center" }
+
+On the right side of the menu window, there is a `DevicePath` which is a
+UEFI-compliant path to the device or file being executed. Depending on the
+file/device type, these paths may be different:
+
+![](/images/menus/one_time_boot.jpeg){ class="center" }
+
+![](/images/menus/one_time_boot3.jpeg){ class="center" }
+
+`DevicePaths` are defined in [UEFI Specification](https://uefi.org/specifications).
+If you are interested in decoding those, read through the specification
+carefully.
+
+### Boot Manager Menu
+
+Boot Manager Menu is an application that lists all bootable options and allows
+one to override the boot path.
+
+Boot Manager Menu is entered with a different key than setup application. It
+may be customized on your platform. The right key to use is always printed on
+the screen in the top-left corner, for example `F11`:
+
+```txt
```

Comment:
Information about entering the menus is also available at the beginning of the documentation. Perhaps a better solution is to indicate access to the menus at the beginning of the document (with additional information or a link to dedicated documentation which will contain a labeled list of keys for all Dasharo products)

Reply:
Yes, but it is just mentioned briefly o the beginning. Here is more detailed description